routes/verifiyToken.js

```diff
@@ -0,0 +1,14 @@
+require('dotenv').config();
+const jsonWebToken = require('jsonwebtoken');
+
+module.exports = function (request, response, next) {
+  const loginToken = request.cookies.loginToken;
+  if (!loginToken) return response.status(401).send('Access Denied');
+  try {
+    const verified = jsonWebToken.verify(loginToken, process.env.TOKEN_SECRET);
+    request.user = verified;
```

Comment:
`user` sounds like an object with multiple user properties. But there is only the user id.
I would make sure this is not misleading:

```suggestion
    request.userId = verified._id;
```

or if you rename `verified` and change the payload to `{ userId: user._id }`:

```suggestion
    request.userId = payload.userId;
```